kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/maps/homework/School.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.collections.adv.maps.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class School {
+    private List<Integer> school = new ArrayList<>();
+    private String name = "";
+
+    public School(String name, int ... school) {
```

Comment:
co to za parametr int ... school? to sie kompiluje w ogole?

Reply:
public School(String name, List<Integer> studentCountInClasses) {this.name = name; this.studentCountInClasses;}